java/ql/src/semmle/code/java/frameworks/android/SharedPreferences.qll

```diff
@@ -0,0 +1,62 @@
+import semmle.code.java.Type
+
+/** Definitions related to `android.content.SharedPreferences`. */
+module SharedPreferences {
```

Comment:
```suggestion
/** Provides classes related to `android.content.SharedPreferences`. */

import java
```
Let's skip the top-level module and update some of the class names to be a bit more specific than the short-hand `Prefs`.
Also, the first import should always be `import java` (which includes `import semmle.code.java.Type`), and the file should have a top-level qldoc.

Reply:
Changed

Reply:
@aschackmull - Sorry this is the first time I work on a module. I'm not sure whether my changes are what you want. Please review.

Thanks,
@luchua-bc

Reply:
Sorry I wasn't clear enough about this file. I'll write something more specific inline.

Reply:
Thanks a lot for the detailed instructions. I've made requested changes.